include/d/d_com_inf_game.h

```diff
@@ -3163,6 +3163,10 @@ inline void dComIfGp_particle_calcMenu() {
     g_dComIfG_gameInfo.play.getParticle()->calcMenu();
 }
 
+inline int dComIfGp_particle_addModelEmitter(dPa_J3DmodelEmitter_c* emitter){
+    return cLs_Addition(g_dComIfG_gameInfo.play.getParticle()->mModelControl, emitter);
+}
```

Comment:
FYI, if you check the debug map for this actor, `d_a_obj_eskbanD.map`, you can see three nested inlines are used here:
```cpp
           10] dComIfGp_particle_addModelEmitter__FP18dPa_modelEmitter_c (func,weak) found in d_a_obj_eskban.o 
            11] addModelEmitter__13dPa_control_cFP18dPa_modelEmitter_c (func,weak) found in d_a_obj_eskban.o 
             12] add__18dPa_modelControl_cFP18dPa_modelEmitter_c (func,weak) found in d_a_obj_eskban.o 
>>> SYMBOL NOT FOUND: cLs_Addition__FP15node_list_classP10node_class
```

So `dComIfGp_particle_addModelEmitter` would call `g_dComIfG_gameInfo.play.getParticle()->addModelEmitter`, then that would call `mModelControl->add`, and that would call `cLs_Addition`.

In some cases like this using the inlines the same way as the debug map doesn't really matter as it doesn't affect the match. But there are also some cases where you have to copy the way they used inlines exactly for the function to match 100% so I usually do that.